src/shared/components/post/metadata-card.tsx

```diff
@@ -76,9 +76,8 @@ export class MetadataCard extends Component<
                             <button
                               className="mt-2 btn btn-secondary text-monospace"
                               onClick={linkEvent(this, this.handleIframeExpand)}
-                              data-tippy-content={i18n.t("expand_here")}
                             >
-                              {this.state.expanded ? "-" : "+"}
+                              {i18n.t("expand_here")}
```

Comment:
Only showing `+` on the button was confusing, so i changed it to show the text directly (not in tooltip). Also `-` was never shown because the button gets hidden when embed is displayed.